### DISCUSSION TITLE
Numbering of dev release

### DISCUSSION THREAD

**Discussion by Delta1977** [emsesp/EMS-ESP32#1990] [General]
There are now more then 3 release from dev34. Is this normal ? Shouldn't every release count up the dev version ?

**Comment by proddy** [comment]
Technically, yes. The proper way would be to bundle the PRs together, say weekly, review them, and push them while incrementing the release version. However, I'm making very small changes daily, and the changes are so minor that they're not really worth mentioning in the release notes or updating the version. For example, the last few had improved German translations in the WebUI and an extra line in the system info API.  If the version increased, it will panic users as the WebUI will mention a new release is available every day. When 3.7.0 finally gets released we'll go back to normal.
